coding_guidelines_c++.md

```diff
@@ -31,22 +32,17 @@ _Note that older code might not conform to this rules and should be fixed when c
         * [Further Indentation and Formatting](#further-indentation-and-formatting)
     * [Stopwatch example](#stopwatch-example)
 - [.clang-format File](#.clang-format-file)
-- [Vera++ profile nest](#vera++-profile-nest)
+- [Vera++ profile nest](#vera-profile-nest)
 
 
 ## Tooling
 
-The code has to compile without warnings (in the default settings of the build infrastructure). We restrict ourselfs to C++03 standard for a larger support of compilers on various cluster systems and supercomputers.
+The code has to compile without warnings (in the default settings of the build infrastructure). We restrict ourselves to C++03 standard for a larger support of compilers on various cluster systems and supercomputers.
 
 The [clang-format](http://clang.llvm.org/docs/ClangFormat.html) tool is build on the clang compile fronted. It allows to pretty print input files in a configurable manner, and has vim and emacs integration. We supply a [.clang-format](#clang-format-file) configuration file to enforce some parts of the coding style. During the code review process we check that there is no difference between the committed files and the formatted version of the committed files:
 
-    clang-format <committed file> > tmp_out.txt
-    diff <committed file> tmp_out.txt > /dev/null
-    if [ $? -ne 0 ]; then
-      # error
-    fi
 
-Developer can benefit from the tool by formatting their changes before issuing a pull request: `clang-format -i <committed file>`. We require clang-format version 3.6 or later.
+Developer can benefit from the tool by formatting their changes before issuing a pull request: `clang-format -i <committed file>`. We require clang-format version 3.6.
```

Comment:
Homebrew provides clang-format 3.7 only right now and no recipes for older versions. I supplied a download link of pre-build binaries for clang-format-3.6 for OS X.